packages/Constants/src/Constants.js

```diff
@@ -0,0 +1,61 @@
+export const align = {
+  TOP: "top",
+  RIGHT: "right",
+  BOTTOM: "bottom",
+  LEFT: "left",
+};
+
+export const color = {
+  BLACK: "black",
+  BLUE: "blue",
+  GREEN: "green",
+  GREY: "grey",
+  ORANGE: "orange",
+  LIGHT_BLUE: "lightBlue",
+  LIGHT_ORANGE: "lightOrange",
+  RED: "red",
+};
+
+export const kind = {
+  DEFAULT: "default",
+  PRIMARY: "primary",
+  SECONDARY: "secondary",
+  DESTRUCTIVE: "destructive",
+  FLAT: "flat",
+  MINOR: "minor",
+  LINK: "link",
+};
+
+export const size = {
+  AUTO: "auto",
+  XSMALL: "xsmall",
+  SMALL: "small",
+  MEDIUM: "medium",
+  LARGE: "large",
+  XLARGE: "xlarge",
+};
+
+export const autoSize = {
+  AUTO: "auto",
+  SMALL: "small",
+  MEDIUM: "medium",
+  LARGE: "large",
+};
+
+export const defaultSize = {
+  SMALL: "small",
+  MEDIUM: "medium",
+  LARGE: "large",
+};
+
+export const limitedSize = {
+  SMALL: "small",
+  MEDIUM: "medium",
+};
+
+export const gridTypes = {
```

Comment:
ohh this are here because the DataGrid and the Table use them correct?

Reply:
Correct!